README.md

```diff
@@ -18,26 +18,26 @@ If you'd like to easily switch between versions of Node, check out [nvm](https:/
 To get started you'll need Yeoman and Gulp installed.
 
 ```
-npm i -g yo gulp
+npm i -g yo@3.1.0 gulp@3.9.1
```

Comment:
Maybe we should add a warning that using other versions can have unexpected results.  I notice a lot of customers skip prerequisites unless there is a warning

Reply:
Addressed in 
52e440f

![image](https://user-images.githubusercontent.com/16601729/126396154-a7ebfaa9-8d39-4197-9036-ff42017cb697.png)